features/financial_assistance/step_definitions/other_questions_page_steps.rb

```diff
@@ -104,6 +104,15 @@
   expect(page).to have_content('Has this person applied for an SSN *')
 end
 
+And(/^the is primary caregiver question should display$/) do
+  expect(page).to have_css('#is_primary_caregiver_no')
+end
+
+Then(/^there should not be a default value$/) do
+  expect(page).to_not have_css('#is_primary_caregiver_no[checked]')
```

Comment:
I see what the intention is here, but let's assert on things that exist rather than asserting that things don't exist

Reply:
Hey @markgoho is this better here? `expect(page).to have_css('#is_primary_caregiver_no:not(:checked)')`
I think the best way of testing this scenario is still to make sure the radio button isn't checked. I'm open to other ideas if you have some.